stylua.toml

```
@@ -0,0 +1,5 @@
+indent_type = "Tabs"
+indent_width = 2
+column_width = 120
+call_parentheses = "NoSingleTable"
+collapse_simple_statement = "FunctionOnly"
```

```suggestion
indent_width = 2
call_parentheses = "NoSingleTable"
collapse_simple_statement = "FunctionOnly"
```

`indent_type = "Tabs"` and `column_width = 120` is the default value